pkg/account/service_test.go

```diff
@@ -197,6 +197,30 @@ func TestUpdate(t *testing.T) {
 			},
 			returnErr: nil,
 		},
+		{
```

Comment:
unit test coverage for `mergo.Merge` changes (verified this test fails, without those merge changes)